src/plugins/messenger/MessengerPreview/components/MessengerButton/MessengerButton.tsx

```diff
@@ -33,9 +33,20 @@ export const getMessengerButton = ({ React, styled }: MessagePluginFactoryProps)
         }
     }));
 
-    const MessengerButton = ({ button, ...props }: IMessengerButtonProps & React.ComponentProps<typeof Button>) => (
-        <Button {...props} dangerouslySetInnerHTML={{__html: getButtonLabel(button)}} />
-    )
+    const MessengerButton = ({ button, ...props }: IMessengerButtonProps & React.ComponentProps<typeof Button>) => {		
+        const buttonType = button.type;
+        const isWebURL = buttonType === "web_url";
+        const ariaLabel = isWebURL ? button.title + " .Opens in new tab" : undefined;
```

Comment:
should be `title + '. Opens'`. Switch the dot with the space.

Reply:
Fixed